# generated by patch-package 6.4.8 on 2022-01-30 09:41:52
#
# command:
#   npx patch-package @changesets/cli
#
# declared package:
#   @changesets/cli: 2.19.0
#
diff --git a/node_modules/@changesets/cli/dist/cli.cjs.dev.js b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
index f01eca6..bf26ef5 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.dev.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
@@ -1142,8 +1142,8 @@ async function run$1(cwd) {
   } = await getPackages.getPackages(cwd);
   const allExistingTags = await git.getAllTags(cwd);
 
-  for (const pkg of packages) {
-    const tag = tool !== "root" ? `${pkg.packageJson.name}@${pkg.packageJson.version}` : `v${pkg.packageJson.version}`;
+  for (const pkg of packages.filter(pkg => ["electron-builder", "electron-updater"].includes(pkg.packageJson.name))) {
+    const tag = (tool !== "root" && pkg.packageJson.name !== 'electron-builder') ? `${pkg.packageJson.name}@${pkg.packageJson.version}` : `v${pkg.packageJson.version}`;
 
     if (allExistingTags.has(tag)) {
       logger.log("Skipping tag (already exists): ", tag);
diff --git a/node_modules/@changesets/cli/dist/cli.cjs.prod.js b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
index d630392..40448f0 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.prod.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
@@ -628,8 +628,8 @@ async function pre(cwd, options) {
 
 async function run$1(cwd) {
   const {packages: packages, tool: tool} = await getPackages.getPackages(cwd), allExistingTags = await git.getAllTags(cwd);
-  for (const pkg of packages) {
-    const tag = "root" !== tool ? `${pkg.packageJson.name}@${pkg.packageJson.version}` : "v" + pkg.packageJson.version;
+  for (const pkg of packages.filter(pkg => ["electron-builder", "electron-updater"].includes(pkg.packageJson.name))) {
+    const tag = (tool !== "root" && pkg.packageJson.name !== 'electron-builder') ? `${pkg.packageJson.name}@${pkg.packageJson.version}` : "v" + pkg.packageJson.version;
     allExistingTags.has(tag) ? logger.log("Skipping tag (already exists): ", tag) : (logger.log("New tag: ", tag), 
     await git.tag(tag, cwd));
   }
diff --git a/node_modules/@changesets/cli/dist/cli.esm.js b/node_modules/@changesets/cli/dist/cli.esm.js
index abaa3c8..c7131b5 100644
--- a/node_modules/@changesets/cli/dist/cli.esm.js
+++ b/node_modules/@changesets/cli/dist/cli.esm.js
@@ -1119,8 +1119,8 @@ async function run$1(cwd) {
   } = await getPackages(cwd);
   const allExistingTags = await getAllTags(cwd);
 
-  for (const pkg of packages) {
-    const tag$1 = tool !== "root" ? `${pkg.packageJson.name}@${pkg.packageJson.version}` : `v${pkg.packageJson.version}`;
+  for (const pkg of packages.filter(pkg => ["electron-builder", "electron-updater"].includes(pkg.packageJson.name))) {
+    const tag$1 = (tool !== "root" && pkg.packageJson.name !== 'electron-builder') ? `${pkg.packageJson.name}@${pkg.packageJson.version}` : `v${pkg.packageJson.version}`;
 
     if (allExistingTags.has(tag$1)) {
       log("Skipping tag (already exists): ", tag$1);
